templates/zerver/help/change-a-users-role.md

```diff
@@ -15,22 +15,37 @@ there is at least one other owner in the organization (consider
 promoting a new owner or [deactivating the
 organization](/help/deactivate-your-organization) instead).
 
-**Changes** Organization owners were introduced in Zulip 3.0; users
-that were marked as administrators in older Zulip instances are
+**Changes**: Organization owners were introduced in Zulip 3.0. Users
+who were marked as administrators in older Zulip instances are
 automatically converted during the upgrade to Zulip 3.0 into owners
 (who have the same permissions as administrators did previously).
```

Comment:
@timabbott Can we drop this paragraph at this point? It refers to 2.x->3.0 migration.

Reply:
Yes, I'll do that.